s4e-web/cypress/page-objects/settings/institution-list.po.ts

```diff
@@ -0,0 +1,37 @@
+import 'cypress-file-upload';
+import {SideNav} from './side-nav.po';
+import {AddInstitutionForm} from './add-institution-form.po';
+
+export namespace InstitutionList {
+  export class PageObject {
+    static getNewBtn = () => cy.get('[data-e2e="addInstitution"]');
+    static getNthListEntry = (i: number) => cy.get(`table[data-e2e="generic-list-data-table"] > tbody > tr[data-e2e="entity-row"]:nth-child(${i + 1})`);
+    static getAllListEntries = () => cy.get(`table[data-e2e="generic-list-data-table"] > tbody > tr`);
+  }
+
+  export function goToAddInstitution() {
+    PageObject.getNewBtn().click();
+    return AddInstitutionForm;
+  }
+
+  export function getSideNav() {
+    return SideNav;
+  }
+
+  export function getEntries() {
+    return PageObject.getAllListEntries();
+  }
+
+  export function deleteEntry(index: number) {
+    PageObject.getAllListEntries().should('have.length', 2);
+    PageObject.getNthListEntry(index).find('[data-e2e="delete"]').click();
+    cy.get('#accept_btn').click();
+    PageObject.getAllListEntries().should('have.length', 1);
```

Comment:
Could you return Page (namspace) on which you can operate after your action (to be consistent with tests)